mu4/scenes/inspector/models/inspectorlistmodel.cpp

```diff
@@ -192,3 +199,26 @@ bool InspectorListModel::isModelAlreadyExists(const AbstractInspectorModel::Insp
     return false;
 }
 
+void InspectorListModel::subscribeOnSelectionChanges()
+{
+#ifdef BUILD_UI_MU4
+    if (!context() || !context()->currentNotation()) {
+        setElementList(QList<Ms::Element*>());
+    }
+
+    context()->currentNotationChanged().onNotify(this, [this]() {
+        m_notation = context()->currentNotation();
```

Comment:
You need to unsubscribe from the previous notation

Reply:
Do I really needed? I thought that there is no dependency between the notification and the notation instance.

Reply:
In the future, there may be several loaded notations at the same time. And there will be a switching of the current (top tab panel).
If you do not unsubscribe, then the inspector will be subscribed to several notations at once, this can lead to oddities.

Reply:
Hmm, idea. We can make an implementation of the "current" notation, which will proxy calls to the real current notation and re-subscribing all notifications when the notation is changed. 

Then consumers will not need to unsubscribe and subscribe

Reply:
Yeah, it'd be great